servercfg/serverconf.go

```diff
@@ -427,7 +428,16 @@ func GetPublicIP() (string, error) {
 	var err error
 
 	iplist := []string{"https://ip.server.gravitl.com", "https://ifconfig.me", "https://api.ipify.org", "https://ipinfo.io/ip"}
+	publicIpService := os.Getenv("PUBLIC_IP_SERVICE")
+	if publicIpService != "" {
+		fmt.Println("User provided public IP service is", publicIpService)
```

Comment:
Do we need to validate that publicIpService is a valid url?

Reply:
I assume if it fails, it would move on to the predefined ones

Reply:
> I assume if it fails, it would move on to the predefined ones

Yes it did in my testing.